phase1/gce/gce.jsonnet

```diff
@@ -77,6 +93,20 @@ function(cfg)
           source_ranges: ["0.0.0.0/0"],
           target_tags: ["%(cluster_name)s-master" % p1],
         },
+        [names.master_internal_firewall_rule]: {
```

Comment:
I looked into it, and at the VM level, the master only has one network interface (aside from `lo` and `docker0`) with its private IP address. I think the public IP address routing happens completely outside of the VM, so passing this flag to kubeadm won't actually change anything.